public/mockServiceWorker.js

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (0.41.0).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```

Comment:
I wonder why we are supposed to not touch this file, is it boilerplate configuration (where does it come from?), you have not changed anything?

Reply:
@michaelrambeau Yep. Do not touch this file. It is generated by running `npx msw init public/ --save`
This file is rarely changed, but whenever a new version of `msw` requires changes in this file, they will update it automatically when we execute `npm install msw`/ `npm update msw`

Reference: https://mswjs.io/docs/getting-started/integrate/browser#setup

Reply:
Thank you for the explanation 👍